Student-Class/src/student/pkgclass/StudentClass.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package student.pkgclass;
+
+/**
+ *
+ * @author Ziying Guo
+ */
+public class StudentClass {
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+       Info friend = new Info();
+       friend.setName("XiaoMi Lei");
+       friend.setAddress("8613 18th Ave");
+       friend.setAge(22);
+       friend.setPhoneNumber("718-256-2333");
+       System.out.println("Name: "+friend.getName()+"(friend)");
+       System.out.println("Address: "+friend.getAddress());
+       System.out.println("Age: "+friend.getAge());
+       System.out.println("Phone Number: "+friend.getPhoneNumber()); 
+       
+       System.out.println();
+       
+       Info family = new Info();
+       family.setName("Dami Laji");
+       family.setAddress("Guangzhou Road, Huanggang Ave");
+       family.setAge(33);
+       family.setPhoneNumber("07505789188");
+       System.out.println("Name: "+family.getName()+"(family)");
+       System.out.println("Address: "+family.getAddress());
+       System.out.println("Age: "+family.getAge());
+       System.out.println("Phone Number: "+family.getPhoneNumber());  
+       
+        System.out.println();

```

Comment:
It is not invalid This use is for separating the 3 people's information. It is a empty line.

Sent from my iPhone

> On May 7, 2015, at 11:35, Raffi Khatchadourian <notifications@github.com> wrote:
> 
> In Student-Class/src/student/pkgclass/StudentClass.java:
> 
> > +       System.out.println("Age: "+friend.getAge());
> > +       System.out.println("Phone Number: "+friend.getPhoneNumber()); 
> > +       
> > +       System.out.println();
> > +       
> > +       Info family = new Info();
> > +       family.setName("Dami Laji");
> > +       family.setAddress("Guangzhou Road, Huanggang Ave");
> > +       family.setAge(33);
> > +       family.setPhoneNumber("07505789188");
> > +       System.out.println("Name: "+family.getName()+"(family)");
> > +       System.out.println("Address: "+family.getAddress());
> > +       System.out.println("Age: "+family.getAge());
> > +       System.out.println("Phone Number: "+family.getPhoneNumber());  
> > +       
> > +        System.out.println();
>  Invalid formatting.
> 
> —
> Reply to this email directly or view it on GitHub.
>